profile/README.md

```diff
@@ -7,17 +7,22 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
 
 Below you can find a selection of Espressif’s open-source projects. Our full repository list can be found [here](https://github.com/orgs/espressif/repositories).
 
-## Project Highlight
+## Upcoming Event: DevCon24
 
 <p align="center">
-    <img src="esp_registry_logo.png">
+    <img src="devcon24.png">
 </p>
   
-###  **ESP Component Registry**
+Get excited and stay tuned for talks on various presentation topics, new product releases, and upgraded solutions at the Espressif Developer Conference!
```

Comment:
I will suggest to change to:

Get excited and stay tuned for various talks and Speakers, new product releases, and upgraded solutions at the Espressif Developer Conference!